src/lib/spinel/spinel.h

```diff
@@ -3416,16 +3416,17 @@ enum
     SPINEL_PROP_IPV6_ML_PREFIX = SPINEL_PROP_IPV6__BEGIN + 2,
 
     /// IPv6 (Unicast) Address Table
-    /** Format: `A(t(6CLLC))`
+    /** Format: `A(t(6CCbb))`
      *
      * This property provides all unicast addresses.
      *
      * Array of structures containing:
      *
      *  `6`: IPv6 Address
      *  `C`: Network Prefix Length (in bits)
-     *  `L`: Valid Lifetime
-     *  `L`: Preferred Lifetime
+     *  `C`: Scope
+     *  `b`: Is Preferred
+     *  `b`: Is Mesh Local
```

Comment:
We generally don't change existing NCP property definition (to ensure backward compatibility).
https://github.com/openthread/openthread/blob/4a0d6b7b6495a1e4e2e03ab7a4f8baa064df4a6e/src/lib/spinel/spinel.h#L293-L311

Suggest defining a new property with new format.

Reply:
Thanks! SGTM